pkg/utils/eventrecorder/eventrecorder.go

```diff
@@ -89,3 +94,16 @@ func findMyPod(cachedK8SClient client.Client) (corev1.Pod, error) {
 	}
 	return pod, err
 }
+
+// Functions used for mocking package
+func InitMockEventRecorder() *events.FakeRecorder {
```

Comment:
Post merge review; a mock recorder could be test files as, I hope, we don't use this in real code.